### ISSUE TITLE
Chose evaluator for importance calculation that is needed for `plot_param_importances`

### ISSUE THREAD

**Issue by Aboussejra** [optuna/optuna-dashboard#224]
# Feature Request

As optuna-dashboard is now : 
the parameter importance is calculated using the base evaluator (https://optuna.readthedocs.io/en/stable/reference/generated/optuna.importance.FanovaImportanceEvaluator.html) which is recommended when using RandomSampler (https://optuna.readthedocs.io/en/stable/reference/generated/optuna.samplers.RandomSampler.html#optuna.samplers.RandomSampler)
If my understanding of the following lines is correct.

https://github.com/optuna/optuna-dashboard/blob/d18d271fce7e1a1b2825c201f942e0d7b8c2ebf9/optuna_dashboard/_importance.py#L54-L75

I think maybe adding some options (CLI arguments or in the dashboard) to modify that could be a good idea? What do you think ?

Best regards,

**Comment by c-bata** [comment]
Thank you your proposal!
Yes, it sounds a good idea. I think adding a select box in the dashboard is better than CLI arguments. I labeled this issue `contribution-welcome` 👍

**Comment by Aboussejra** [comment]
Okay, I am presenting this dashboard at my company this afternoon, I hope they like it, if they do and want to use it, maybe they will permit me to contribute :)

**Comment by c-bata** [comment]
I'm not sure but I confirmed that the above command works on my environment.

```
$ docker run -it --rm -p 8080:8080 -v `PWD`:/app -w /app optuna-dashboard sqlite:///db.sqlite3
```

https://github.com/optuna/optuna-dashboard#using-an-official-docker-image

**Comment by Aboussejra** [comment]
On what OS, I tried to run this on linux, maybe that is an issue.

**Comment by c-bata** [comment]
Hmm.... I use macOS.

**Comment by Aboussejra** [comment]
Yeah, I will search a bit more later :), sorry I am not an expert on docker and those things ahah
